src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
@vexx32, your last commit had 2 failures in `PowerShell-CI-linux`
Test-Connection.Ping.Force IPv4 with explicit PingOptions
```powershell
Expected $null or empty, but got True.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 107
107:                 $result1[0].Options.DontFragment | Should -BeNullOrEmpty
```
Test-Connection.MTUSizeDetect.MTUSizeDetect works
```powershell
Expected the actual value to be greater than 0, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 221
221:             $result.MtuSize | Should -BeGreaterThan 0
```